kvbc/src/kvbc_adapter/v4blockchain/app_state_adapter.cpp

```diff
@@ -15,6 +15,7 @@
 #include "kvbc_adapter/v4blockchain/app_state_adapter.hpp"
 #include "v4blockchain/detail/blocks.h"
 #include "v4blockchain/detail/detail.h"
+#include "assertUtils.hpp"
```

Comment:
Is this include really required? I see ConcordAssert() from this header file already being used at line 43.